java-compiler-testing/pom.xml

```diff
@@ -125,7 +125,12 @@
           <failIfNoTests>true</failIfNoTests>
         </configuration>
       </plugin>
-
+      
+      <plugin>
+          <groupId>org.pitest</groupId>
+          <artifactId>pitest-maven</artifactId>
```

Comment:
Do we need to put this in the pluginManagement in the pom.xml in the root of this repo as well? I then usually put the version in the properties block there so dependabot can update it when new versions come out.

Reply:
Add the plugin to build/plugins in your pom.xml

```
<plugin>
    <groupId>org.pitest</groupId>
    <artifactId>pitest-maven</artifactId>
    <version>LATEST</version>
</plugin>

```
That’s it, you’re up and running. 


Mentioned on their website

Reply:
Yeah, we want to freeze the version though, otherwise if they have a breaking change, builds will randomly break

Reply:
True, first we need to look for a stable version then make changes

Reply:
https://mvnrepository.com/artifact/org.pitest/pitest 

Look into the above link latest version is 1.10 there are no vulnerabilities in any of version maybe you can add the right version

Reply:
1.10.3 should be good! Do you want to add it to the parent POM? If not I can take a look later :-)

Reply:
done also can you email me where to communicate with you

Reply:
Right now, w.r.t. this project, best place is probably the discussions tab (I don't really have any other social stuff I am active on unfortunately. Might set a Gitter up or IRC tomorrow though.

Reply:
Going to cherry pick your changes for this from both the PRs you opened into a new PR and merge it from there so it can be applied as one change, hope that is all okay.